clang/test/Headers/__cpuidex_conflict.c

```diff
@@ -3,7 +3,10 @@
 // RUN: %clang_cc1 %s -ffreestanding -fms-extensions -fms-compatibility \
 // RUN:  -fms-compatibility-version=19.00 -triple x86_64-pc-windows-msvc -emit-llvm -o -
 // %clang_cc1 %s -ffreestanding -triple x86_64-w64-windows-gnu -fms-extensions -emit-llvm -o -
-// RUN: %clang_cc1 %s -ffreestanding -fopenmp -fopenmp-is-target-device -aux-triple x86_64-unknown-linux-gnu
+//
+// TODO(boomanaiden154)
```

Comment:
TODO with a name in it?  Typically we just do a 'fixme', particularly awkward to call out other folks in the TODO.

Reply:
Thanks for the review! 

@boomanaiden154 specifically suggested this [here](https://github.com/llvm/llvm-project/pull/121839#discussion_r1925920983), but I'm happy to change it to FIXME if you prefer.

Reply:
Rather than a `TODO` and the name, a FIXME + the issue should be sufficient.  Names like that in source are one of those things that age poorly :)

Reply:
sure, will do!

Reply:
fixed in https://github.com/llvm/llvm-project/pull/121839/commits/0ee8d8cd4b3d61fc6cf6e740a54dd5a23c921048

Reply:
fixed in https://github.com/llvm/llvm-project/commit/0ee8d8cd4b3d61fc6cf6e740a54dd5a23c921048!